nextjs/pages/index.js

```diff
@@ -1,17 +1,108 @@
-const Home = () => (
-  <section className="main">
-    <div className="terminal-alert terminal-alert-primary">Post List</div>
-    <figure>
-      <img src="https://http.cat/102.jpg" />
-    </figure>
-    <div className="progress-bar progress-bar-show-percent">
-      <div
-        className="progress-bar-filled"
-        style={{ width: '5%' }}
-        data-filled="Loading 5%"
-      ></div>
-    </div>
-  </section>
-);
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import { getPosts } from '../store/getPosts/thunks';
+import { selectGetPosts } from '../store/getPosts/selector';
+import PostItem from '../components/PostItem';
+
+const Home = () => {
+  const dispatch = useDispatch();
+  const router = useRouter();
+  const { date, page } = router.query;
+  const { isLoading, response } = useSelector(selectGetPosts);
+
+  useEffect(() => {
+    getPostByPage({ date, page });
+  }, []);
+
+  const getPostByPage = ({ page, date }) => {
+    dispatch(
+      getPosts({
+        date: date || 'all',
+        page: page || 1,
+      }),
+    );
+  };
+
+  return (
+    <section className="main">
+      <>
```

Comment:
Không cần thẻ empty nhỉ
```suggestion
    <section className="main">
```

Reply:
vâng ạ